src/background.js

```diff
@@ -1,13 +1,17 @@
-'use strict';
+import Aria2 from 'aria2';
+
+function validateUrl(value) {
+    return /^(?:(?:(?:https?|ftp):)?\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))(?::\d{2,5})?(?:[/?#]\S*)?$/i.test(value);
```

Comment:
@gautamkrishnar can you provide a regex in replacement of this one as it seems to be having some issues.
The regex test can be found at [regex101](https://regex101.com/r/0fPihZ/1).
And the lint report is:
![image](https://user-images.githubusercontent.com/37300847/86231433-30778980-bbb0-11ea-9c3e-dbf2f7c12842.png)

Reply:
@149segolte it is taken from https://stackoverflow.com/questions/8667070/javascript-regular-expression-to-validate-url

Feel free to use any. I think checking http/https and :// would be enough. You can use any other regex mentioned in the Stackoverflow.